tests/plugins/crop_generators/test_autoencoder_crop_generator.py

```diff
@@ -62,13 +61,29 @@ def metadata(out_dir, make_struct):
     meta.splitMetadata(train_rule, val_rule)
     return meta
 
+@pytest.fixture(scope="function")
+def locations(out_dir, metadata, config, make_struct):
+    meta = metadata.data.iloc[0]
+    path = os.path.abspath(os.path.join(config["paths"]["locations"], meta["Metadata_Plate"]))
+    os.makedirs(path, exist_ok=True)
+    path = os.path.join(path,
+        "{}-{}-{}.csv".format(meta["Metadata_Well"],
+        meta["Metadata_Site"],
+        "Nuclei"))
+    locations = pd.DataFrame({
+        "Nuclei_Location_Center_X": np.random.randint(0, 128, 10),
+        "Nuclei_Location_Center_Y": np.random.randint(0, 128, 10)
+    })
+    locations.to_csv(path, index=False)
+
 
 @pytest.fixture(scope="function")
-def dataset(metadata, out_dir, config, make_struct):
+def dataset(metadata, out_dir, config, make_struct, locations):
```

Comment:
Locations doesn't seem to be used in this function.

Reply:
They have to be initilized in order to make ```dset.prepare_training_locations()``` call work.